shared/chat/conversation/info-panel/settings/index.tsx

```diff
@@ -134,7 +142,7 @@ const SettingsPanel = (props: SettingsPanelProps) => {
             {canDeleteHistory && (
               <Kb.Button
```

Comment:
were you also going to change the 2 below actions into buttons?

Reply:
Oh yes, will do that now!